python/parquet_tools/csv2pq_utils.py

```diff
@@ -0,0 +1,111 @@
+# *****************************************************************************
+# *                       c s v 2 p q _ u t i l s . p y                       *
+# *****************************************************************************
+#
+# This file is part of parquet_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+The csv2pq_utils.py module contains utility functions used by the csv2pq
+command and its assiciated python modules.
+
+This module provides various functions to make life somewhat easier:
+ePrint    - print a message to stderr.
+Fatal     - print a numbered message to stderr and exit.
+getVal    - get the next value from the command line argument vector.
+traceBack - print a stack traceback.
+"""
+
+from __future__ import print_function
+
+import sys
+import traceback
+
+
+# *****************************************************************************
+# *                                e P r i n t                                *
+# *****************************************************************************
+
+def ePrint(*args, **kwargs):
```

Comment:
NO! This is a command line utility and like all CLI's the output has to go to the command issuer (a human) in an understandable format just like all other UNIX commands! Got it?

Reply:
Plus, think about what you are saying. If that is true then I can't use argparse as it is because I would need to vector all error output to a logger. Come on, this is crazy!

Reply:
If you use `logging` you have full control over what happens to the log messages. You can configure it to print them all or configure it to hide them. This is handy if you are implementing `--verbose` or `--quiet` command line switches and have used different log levels in the system.

Reply:
I'm not saying that _all_ output should go to `logging`, I think for "normal" output you do use regular`print()` in few places, and I don't propose to replace those. Use cases for `ePrint()` that I saw in your code look like "non-normal" or logging type of output. For those I think it's natural to use `logging`.
And `argparse` should print it's errors to `stderr` directly without going through logging, it's also for user consumption.

Reply:
Well, every single case I looked at was user oriented information. Either an error message or a comfort note that someone sitting at a terminal would see that something was actually happening.